doc/userguide/rules/tls-keywords.rst

```diff
@@ -17,10 +31,24 @@ Examples::
 
 ``tls.cert_subject`` can be used as ``fast_pattern``.
 
-``tls.cert_subject`` replaces the previous keyword name: ``tls_cert_subject``. You may continue
-to use the previous name, but it's recommended that rules be converted to use
+**Note:** ``tls.cert_subject`` replaces the previous keyword names: ``tls_cert_subject`` and ``tls.subject``. 
+You may continue to use the previous names, but it's recommended that rules be converted to use
 the new name.
```

Comment:
nit: replace "new name" with "new one", to prevent having "name" so many times on this paragraph xD

(same can be done in the other similar note)